neqo-crypto/Cargo.toml

```diff
@@ -22,7 +22,7 @@ neqo-common = { path = "../neqo-common" }
 
 [build-dependencies]
 # Sync with https://searchfox.org/mozilla-central/source/Cargo.lock 2024-02-08
-bindgen = { version = "0.69", default-features = false, features = ["runtime"] }
+bindgen = { version = "0.70", default-features = false, features = ["runtime"] }
```

Comment:
mozilla-central is still at `0.69`:

https://searchfox.org/mozilla-central/rev/527d691a542ccc0f333e36689bd665cb000360b2/Cargo.lock#443-459